.github/workflows/generate-command-reference.yml

```diff
@@ -0,0 +1,55 @@
+name: Automatically generate command reference topics
+on:
+  push:
+  workflow_dispatch:
+
+env:
+  AMC_COMMAND_REFERENCE_DIR: "./reference/cmd-ref/amc"
+  APPLIANCE_COMMAND_REFERENCE_DIR: "./reference/cmd-ref/appliance"
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-24.04
+    steps:
+    - name: Determine base version
+      id: base_version
+      run: |
+        echo "value=$(cat .base_version)" >> "$GITHUB_OUTPUT"
+    - name: Setup Anbox Cloud
+      uses: canonical/anbox-cloud-github-action@39490d442cc0921fe614f7044b9f3dc8bd89725c
+      with:
+        channel: ${{ steps.base_version.outputs.value }}/stable
+    - name: Checkout repo
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
+    - name: Delete existing files - amc reference
```

Comment:
Since cleanup, generation and outputs are trivial steps can we combine these into one ? e.g

```yaml
    - name: Generate AMC command reference documentation
      id: amc_command_reference_dir
      run: |
         rm -rf $AMC_COMMAND_REFERENCE_DIR/*
         amc generate documentation -f markdown -d $AMC_COMMAND_REFERENCE_DIR
         echo "value=$AMC_COMMAND_REFERENCE_DIR" >> "$GITHUB_OUTPUT"
```

we can do the same for appliance command reference.